tests/cxx/update_comments_long-d1.h

```diff
@@ -0,0 +1,27 @@
+//===--- update_comments_long-d1.h - test input file for iwyu -------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// This header exists to verify that --update_comments=long works
```

Comment:
This comment is no longer correct. If possible, I think you should:

* Remove the `update_comments*.h" header files
* Rename the new `update_comments*.cc` test cases to `comment_style_update*.cc`
* Reuse the `comment_style` headers for these tests

That way it's a little clearer how the two are related.

Reply:
I guess the trick is to avoid IWYU diagnostics for the update tests, but that's possible by including `comment_style-d1.h` and only using the `Foo::bar` symbol.